sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StatefulProcessorHandleImplBase.scala

```diff
@@ -39,7 +39,7 @@ abstract class StatefulProcessorHandleImplBase(
         timeMode.toString)
     }
 
-    if (currState < INITIALIZED || currState >= TIMER_PROCESSED) {
+    if (currState < PRE_INIT || currState >= TIMER_PROCESSED) {
```

Comment:
Is this change related ?

Reply:
Yeah it is, since we call `registerTimer` in the PRE_INIT phase to add the columnFamilySchemas on the driver side.